kahuna/public/js/components/gr-collections-panel/gr-collections-panel-node.html

```diff
@@ -19,31 +19,47 @@
         </button>
 
         <a class="node__name flex-spacer"
-           ui:sref="search.results({query: (node.data.name)})">
+           ui:sref="search.results({query: '~'+node.data.content.pathId})">
             {{node.data.name}}
         </a>
 
+
         <button class="node__action clickable" type="button"
-                ng:if="ctrl.deletable"
-                ng:click="ctrl.remove()">
+                ng:if="ctrl.deletable && ctrl.editing"
+                ng:click="ctrl.remove()"
+                gr:tooltip="Delete this collection">
             <gr-icon-label gr-icon="delete"></gr-icon-label>
         </button>
 
         <button class="node__action clickable"
                 type="button"
-                ng:click="active = !active">
+                ng:if="ctrl.editing"
+                ng:click="active = !active"
+                gr:tooltip="Add a new sub-collection">
             <gr-icon>add_box</gr-icon>
         </button>
-    </div>
 
-    <form ng:show="active" ng:submit="ctrl.addChild(childName).then(clearForm);">
-        <input type="text" required ng:model="childName" />
-        <button type="submit">
-            <gr-icon-label gr-icon="check"></gr-icon-label>
-        </button>
-        <button type="button" ng:click="clearForm()" title="Close">
-            <gr-icon-label gr-icon="close"></gr-icon-label>
+        <button class="node__action clickable"
+                ng:if="!ctrl.editing && ctrl.hasImagesSelected && !ctrl.saving"
+                ng:click="ctrl.addImagesToCollection()"
+                gr:tooltip="Add selected images to this collection">
+            <gr-icon>add_to_photos</gr-icon>
         </button>
+        <span ng:if="ctrl.saving">Saving…</span>
+    </div>
+
+    <form class="node__add-child" ng:show="active" ng:submit="ctrl.addChild(childName).then(clearForm);">
+        <input class="node__add-child-input" type="text" required ng:model="childName"
+               placeholder="olympics, pantaloons, summer2015…" />
```

Comment:
Is this placeholder a bit confusing? On labels atm you can submit 'x, y, z' to create 3 labels 'x', 'y', 'z' but on collections you would create one new collection called 'x, y, z' .